040-Data-operations/160-image-transformations.mdx

```diff
@@ -22,149 +26,122 @@ https://us-east-1.storage.xata.sh/transform/rotate=180,height=100,format=webp/4u
   options={{ rotate: 180, height: 100 }}
 />
 
-## Record based transformations
-
-When querying a `file` of `file[]` column from a record, you can make a direct transformation on any of the files it contains.
+## SDK transformation helpers
```

Comment:
@snide we could delete this entire section

Reply:
This is fine, but needs to retain how to pull `metaDataUrl` from the client directly, rather than needing to fetch it manually.